etc/profile-m-z/mediathekview.profile

```diff
@@ -27,6 +29,7 @@ include disable-interpreters.inc
 include disable-programs.inc
 include disable-xdg.inc
 
+whitelist ${HOME}/.mediathek3
```

Comment:
Suggestion: add this just above the whitelist
`mkdir ${HOME}/.mediathek3`

Reply:
We should also (no)blacklist this path.

Reply:
@rusty-snake We already do so...

Reply:
Ohh yes, but then we should whitelist all noblacklisted paths, otherwise it is a bit useless to noblacklist them.

Reply:
I'll do:

add blacklist ${HOME}/.mediathek3 in disable-programs.inc

noblacklist ${HOME}/.mediathek3